haystack/nodes/prompt/prompt_node.py

```diff
@@ -681,6 +681,27 @@ def get_predefined_prompt_templates() -> List[PromptTemplate]:
             name="translation",
             prompt_text="Translate the following context to $target_language. Context: $documents; Translation:",
         ),
+        PromptTemplate(
+            name="think-step-by-step",
```

Comment:
What if we call this template zero-shot-react? We could also provide a few-shot-react with examples. Let's talk offline.

Reply:
I have no strong opinion on the template name. Happy to discuss alternatives. 👍

Reply:
We decided to call this template `zero-shot-react`. In another PR, we could later add a template `few-shot-react`.

Reply:
ReAct to honour the ideas from https://arxiv.org/abs/2210.03629